packages/react-html/package.json

```diff
@@ -29,7 +29,8 @@
     "@shopify/useful-types": "^2.0.1",
     "@types/multistream": "^2.1.1",
     "multistream": "^2.1.1",
-    "tslib": "^1.9.3"
+    "tslib": "^1.9.3",
+    "serialize-javascript": "^2.1.0"
```

Comment:
This was a strange one to be missing.

Reply:
Why?

Reply:
This library uses it heavily so I'm surprised the library works without it listed. I think its because `react-serialize` has it.